src/timer/CountDownTimer.jsx

```diff
@@ -0,0 +1,73 @@
+import React, {useEffect, useState} from 'react';
```

Comment:
we should look in this file for behaviors we need to replicate in this timer. The one I mentioned earlier was the ping interval provided by the backend (how often the timer should sync)  I also see some accessibility features in there we'll need to cover.
https://github.com/edx/edx-proctoring/blob/master/edx_proctoring/static/proctoring/js/views/proctored_exam_view.js#L156

Reply:
The ping interval functionality will be a part of the proctored exam sprint if you have not objection

Reply:
we can definitely bump this to another PR, but if we want to release the completed timed exam feature to a beta partner I think we will need this to maintain our current integrity standards for timed exams.

Can you make this into a ticket so we can tackle it early within that sprint?

Reply:
Zach, polling feature was added, please check in https://github.com/raccoongang/frontend-lib-special-exams/pull/5/commits/15abafdbed9d3ca1267e854a2b2c9b3b15b89d2f